test/end-to-end/vehicle.js

```diff
@@ -218,6 +218,36 @@ test('vehicle permissions', async(t) => {
   t.is(response.meta.requestId.length, 36);
 });
 
+test('vehicle subscribe and unsubscribe - success', async(t) => {
+  const webhookId = 'fdc63623-cd65-4a7f-8ef4-44985f9d7355';
```

Comment:
Gurpreet made environment variables in Travis for testing webhooks! 

`E2E_SMARTCAR_AMT` and `E2E_SMARTCAR_WEBHOOK_ID`